### ISSUE TITLE
Running 123PV model simulation doesn't show output plots

### ISSUE THREAD

**Issue by poorva1209** [GRIDAPPSD/GOSS-GridAPPS-D#802]

**Comment by bibiraju** [comment]
@afisher1 I don't see any regulators or switches for ieee123pv in it's dictionary. Is it how it is supposed to be? The Viz is unable to plot power due to this.

**Comment by afisher1** [comment]
@bibiraju  you are correct. It should be identical to ieee123 model with extra objects. @Jeffrey-Simpson do you know why the ieee123pv model is missing the regulators and switches that should be on the system?

**Comment by Jeffrey-Simpson** [comment]
Hello,

I converted from a model provided by Fei and here is her explination:

I removed the devices on purpose, in order to clearly demonstrate the impact of our DER control algorithm on distribution voltages. If regulators and capacitors exist, it is hard to differentiate the voltage regulation performance between these legacy control devices and our DER control. For the switches, it doesn’t matter with our app. The existence of switches doesn’t have any actual role for our app, so we just removed the switches for simplicity.

Thanks,
Fei
